src/components/login-page/PrivateRoute.jsx

```diff
@@ -0,0 +1,9 @@
+// import React from 'react';
+import { Navigate, Outlet } from 'react-router-dom';
+import { useAuth } from '../../utils/constants';
+
+export default function PrivateRoute() {

```

Comment:
my personal preference is to stay away from default exports and only use named exports. that way it will be obvious when a function changes name